PATCH-FIX-UPSTREAM https://github.com/google/google-gdata/pull/725

Sort input file list

in order to make builds reproducible.
See https://reproducible-builds.org/ for why this is good.

Index: libgoogle-data-mono-1.4.0.2/Makefile
===================================================================
--- libgoogle-data-mono-1.4.0.2.orig/Makefile
+++ libgoogle-data-mono-1.4.0.2/Makefile
@@ -55,20 +55,20 @@ samples =	\
 
 all: $(ALLLIBS) tests $(samples) $(PKGCONFIG_FILES)
 
-core_sources = $(wildcard src/core/*.cs) $(wildcard version/*.cs)
-extensions_sources = $(wildcard src/extensions/*.cs) $(wildcard version/*.cs)
-gbase_sources = $(wildcard src/gbase/*.cs) $(wildcard version/*.cs)
-gcodesearch_sources = $(wildcard src/gcodesearch/*.cs) $(wildcard version/*.cs)
-gspreadsheet_sources = $(wildcard src/gspreadsheet/*.cs) $(wildcard version/*.cs)
-gcalendar_sources =  $(wildcard src/gcalendar/*.cs) $(wildcard version/*.cs)
-gacl_sources = $(wildcard src/gacl/*.cs) $(wildcard version/*.cs)
-gapps_sources = $(wildcard src/gapps/*.cs) $(wildcard version/*.cs)
-gphotos_sources = $(wildcard src/gphotos/*.cs) $(wildcard version/*.cs)
-gdocuments_sources = $(wildcard src/gdocuments/*.cs) $(wildcard version/*.cs)
-gcontact_sources = $(wildcard src/gcontacts/*.cs) $(wildcard version/*.cs)
-youtube_sources = $(wildcard src/youtube/*.cs) $(wildcard version/*.cs)
-health_sources = $(wildcard src/ghealth/*.cs) $(wildcard version/*.cs)
-blogger_sources = $(wildcard src/blogger/*.cs) $(wildcard version/*.cs)
+core_sources = $(sort $(wildcard src/core/*.cs) $(wildcard version/*.cs))
+extensions_sources = $(sort $(wildcard src/extensions/*.cs) $(wildcard version/*.cs))
+gbase_sources = $(sort $(wildcard src/gbase/*.cs) $(wildcard version/*.cs))
+gcodesearch_sources = $(sort $(wildcard src/gcodesearch/*.cs) $(wildcard version/*.cs))
+gspreadsheet_sources = $(sort $(wildcard src/gspreadsheet/*.cs) $(wildcard version/*.cs))
+gcalendar_sources =  $(sort $(wildcard src/gcalendar/*.cs) $(wildcard version/*.cs))
+gacl_sources = $(sort $(wildcard src/gacl/*.cs) $(wildcard version/*.cs))
+gapps_sources = $(sort $(wildcard src/gapps/*.cs) $(wildcard version/*.cs))
+gphotos_sources = $(sort $(wildcard src/gphotos/*.cs) $(wildcard version/*.cs))
+gdocuments_sources = $(sort $(wildcard src/gdocuments/*.cs) $(wildcard version/*.cs))
+gcontact_sources = $(sort $(wildcard src/gcontacts/*.cs) $(wildcard version/*.cs))
+youtube_sources = $(sort $(wildcard src/youtube/*.cs) $(wildcard version/*.cs))
+health_sources = $(sort $(wildcard src/ghealth/*.cs) $(wildcard version/*.cs))
+blogger_sources = $(sort $(wildcard src/blogger/*.cs) $(wildcard version/*.cs))
 test_sources = $(wildcard src/unittests/*.cs) $(wildcard src/unittests/gbase/*.cs) $(wildcard src/unittests/gapps/*.cs)\
 				 $(wildcard src/unittests/spreadsheets/*.cs) $(wildcard src/unittests/youtube/*.cs)  $(wildcard src/version/*cs)\
 				 $(wildcard src/unittests/core/*.cs)  $(wildcard src/unittests/picasa/*.cs)
